views/forms/UserProfileForm/UserProfileFormFields.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import TextInputTopLabel from '../TextInputTopLabel'
+
+import {
+  borderRadius,
+  lineHeight,
+  fontSize,
+  EMAIL_REGEX,
+} from '../../../constants'
+
+import './UserProfileFormFields.css'
+
+const UserProfileFormFields = ({ errors, control }) => {
+  return (
+    <div className="UserProfileFormInputs_box">
+      <TextInputTopLabel
+        control={control}
+        inputLabel={{
+          sx: {
+            color: 'text.primary',
+            fontWeight: 500,
+            lineHeight: lineHeight[21],
+            fontSize: fontSize[18],
+          },
+        }}
+        sx={{
+          mb: '20px',
+        }}
```

Comment:
should these styles be applied to the `TextInputTopLabel` component so we don't need to add them every time?

Reply:
[0730f7e](https://github.com/AMP-SCZ/dpdash/pull/478/commits/0730f7e99359b47d86b078fb3e0ad15f74bb8fcb)